### ISSUE TITLE
Uncaught TypeError: Cannot read property 'disconnect' of undefined

### ISSUE THREAD

**Issue by placrosse** [justmoon/relative-numbers#9]
[Enter steps to reproduce below:]

1. close almost any tab in any pane via any method

**Atom Version**: 1.3.2
**System**: Mac OS X 10.11.2
**Thrown From**: [relative-numbers](https://github.com/justmoon/relative-numbers) package, v0.4.0

### Stack Trace

Uncaught TypeError: Cannot read property 'disconnect' of undefined

```
At /Users/paul/.atom/packages/relative-numbers/lib/line-number-view.coffee:50

TypeError: Cannot read property 'disconnect' of undefined
    at /Users/paul/.atom/packages/relative-numbers/lib/line-number-view.coffee:50:16
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:245:27)
    at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:31:58)
    at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:455:18)
    at atom-tabs.TabBarView.onClick (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:695:17)
```

### Commands

```
     -8:02.5.0 vim-mode:search (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -8:02.4.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -8:01.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -8:01.3.0 blur (atom-text-editor.editor.mini.is-focused)
 20x -8:00.4.0 vim-mode:repeat-search (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
 13x -7:40.1.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:46.2.0 core:undo (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
     -0:45.4.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
     -0:44.1.0 vim-mode:move-to-start-of-file (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:41.5.0 vim-mode:move-to-line (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:40.3.0 vim-mode:move-to-start-of-file (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
  2x -0:38.8.0 blur (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:07.9.0 ex-mode:open (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:07.8.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:05.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  8x -0:05.7.0 blur (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autosave",
      "language-clojure",
      "language-coffee-script",
      "language-csharp",
      "language-go",
      "language-less",
      "language-objective-c",
      "language-perl",
      "language-php",
      "language-ruby",
      "language-ruby-on-rails",
      "language-python",
      "language-sass",
      "metrics",
      "release-notes",
      "package-generator",
      "spell-check",
      "styleguide",
      "update-package-dependencies",
      "welcome",
      "autocomplete-atom-api",
      "autocomplete-html"
    ]
  },
  "relative-numbers": {}
}
```

### Installed Packages

```coffee
# User
browser-plus, v0.0.54
ex-mode, v0.7.0
git-diff-details, v0.21.0
language-rust, v0.4.5
language-scala, v1.1.4
linter, v1.11.3
linter-rust, v0.3.0
minimap, v4.19.0
minimap-git-diff, v4.1.8
project-manager, v2.6.5
racer, v0.17.2
relative-numbers, v0.4.0
remember-file-positions, v0.2.1
rust-api-docs-helper, v0.5.1
sync-settings, v0.6.0
vim-mode, v0.64.0
vim-surround, v0.8.1

# Dev
No dev packages
```

**Comment by yriveiro** [comment]
[Enter steps to reproduce below:]

1. close a tab.

**Atom Version**: 1.3.2
**System**: Mac OS X 10.10.5
**Thrown From**: [relative-numbers](https://github.com/justmoon/relative-numbers) package, v0.4.0


### Stack Trace

Uncaught TypeError: Cannot read property 'disconnect' of undefined

```
At /Users/yriveiro/.atom/packages/relative-numbers/lib/line-number-view.coffee:50

TypeError: Cannot read property 'disconnect' of undefined
    at /Users/yriveiro/.atom/packages/relative-numbers/lib/line-number-view.coffee:50:16
    at Emitter.module.exports.Emitter.emit (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at TextEditor.module.exports.TextEditor.destroyed (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/text-editor.js:245:27)
    at TextEditor.module.exports.Model.destroy (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/model.js:31:58)
    at Pane.module.exports.Pane.destroyItem (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/pane.js:455:18)
    at atom-tabs.TabBarView.onClick (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:695:17)
```

### Commands

```
 17x -2:04.0 vim-mode:move-left (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -2:01.3.0 vim-mode:change (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -1:59.6.0 vim-mode:move-to-last-character-of-line (atom-text-editor.editor.vim-mode.is-focused.operator-pending-mode)
     -1:55.1.0 core:backspace (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -1:52 core:save (atom-text-editor.editor.vim-mode.is-focused.insert-mode.autocomplete-active)
  8x -1:42 core:move-down (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
  8x -1:40.5.0 core:backspace (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -1:38.2.0 core:move-down (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
  6x -1:38 core:backspace (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
  2x -1:35.6.0 core:save (atom-text-editor.editor.vim-mode.is-focused.insert-mode.autocomplete-active)
  2x -1:22.2.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-seti-syntax.theme-atom-dark-ui)
     -1:18.3.0 settings-view:open (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:18.2.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-seti-syntax.theme-atom-dark-ui)
  2x -1:14.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
  2x -1:12.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -0:10.4.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-seti-syntax.theme-atom-dark-ui)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autocomplete",
      "git-tab-status",
      "hyperclick",
      "wrap-guide",
      "autocomplete-php",
      "php-cs-fixer"
    ],
    "themes": [
      "atom-dark-ui",
      "seti-syntax"
    ],
    "projectHome": "/Users/yriveiro/Development/github/"
  },
  "relative-numbers": {}
}
```

### Installed Packages

```coffee
# User
atom-autocomplete-php, v0.18.4
atom-beautify, v0.28.19
atom-cli-diff, v1.1.0
atom-html-preview, v0.1.13
atom-jade, v0.3.0
atom-pair, v2.0.10
autocomplete-paths, v1.0.2
autocomplete-phpunit, v1.0.2
color-picker, v2.0.14
color-tabs, v0.1.8
docblockr, v0.7.3
file-icons, v1.6.13
git-diff-details, v0.21.0
go-plus, v3.5.2
highlight-line, v0.11.0
inline-messenger, v1.2.8
language-rust, v0.4.5
linter, v1.11.3
linter-jsonlint, v1.1.4
linter-rust, v0.3.0
linter-shellcheck, v1.1.0
local-history, v3.2.3
markdown-format, v2.5.1
minimap, v4.19.0
minimap-bookmarks, v0.2.0
minimap-git-diff, v4.1.8
multi-wrap-guide, v0.23.4
phpunit, v1.2.0
phpunit-snippets, v0.1.0
pigments, v0.19.5
pretty-json, v0.4.1
relative-numbers, v0.4.0
rust-api-docs-helper, v0.5.1
seti-syntax, v0.4.1
synced-sidebar, v0.4.3
travis-ci-status, v0.18.0
vim-mode, v0.64.0
wakatime, v5.0.2

# Dev
No dev packages
```

**Comment by tjstankus** [comment]
+1

Steps to reproduce:

1. Close editor pane via cmd-w

**Atom Version**: 1.3.2
**System**: Mac OS X 10.11.2
**Thrown From**: [relative-numbers](https://github.com/justmoon/relative-numbers) package, v0.4.0


### Stack Trace

Uncaught TypeError: Cannot read property 'disconnect' of undefined

```
At /Users/tjs/.atom/packages/relative-numbers/lib/line-number-view.coffee:50

TypeError: Cannot read property 'disconnect' of undefined
    at /Users/tjs/.atom/packages/relative-numbers/lib/line-number-view.coffee:50:16
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:245:27)
    at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:31:58)
    at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:455:18)
    at Pane.module.exports.Pane.destroyActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:433:12)
    at Workspace.module.exports.Workspace.destroyActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:641:35)
    at Workspace.module.exports.Workspace.destroyActivePaneItemOrEmptyPane (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:680:21)
    at atom-workspace.commandRegistry.add.core:close (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:170:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:21.3.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -1:20.8.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -1:19.6.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -1:18.7.0 vim-mode:move-left (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -1:18.3.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -1:16.3.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -1:15.2.0 vim-mode:delete-right (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -1:14.3.0 core:save (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
 21x -1:13.6.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
  8x -1:09.4.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
 79x -1:08.2.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
 11x -0:26.1.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
 74x -0:23.8.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:11.4.0 vim-mode:move-to-start-of-file (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:00.9.0 core:close (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:00.9.0 blur (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "base16-tomorrow-dark-theme"
    ],
    "openEmptyEditorOnStart": false,
    "projectHome": "/Users/tjs/code"
  }
}
```

### Installed Packages

```coffee
# User
asciidoc-preview, v0.6.0
language-asciidoc, v0.12.0
project-manager, v2.6.5
relative-numbers, v0.4.0
ruby-test, v0.9.17
vim-mode, v0.64.0

# Dev
No dev packages
```

**Comment by dvcrn** [comment]
Giving my +1 here. Same problem since last update on closing pane /t abs

**Comment by blacksky86** [comment]
Yeah, same problem here +1

**Comment by paradox460** [comment]
+1

**Comment by r2qokk** [comment]
+1

**Comment by crshd** [comment]
This has been fixed in #11, just needs to be pulled.
